pkg/bufferdecoder/protocol.go

```diff
@@ -16,14 +16,15 @@ const (
 	SendVfsRead
 )
 
-// PLEASE NOTE, YOU MUST UPDATE THE DECODER IF ANY CHANGE TO THIS STRUCT IS DONE.
-
-// Context struct contains common metadata that is collected for all types of events
-// it is used to unmarshal binary data and therefore should match (bit by bit) to the `context_t` struct in the ebpf code.
-// NOTE: Integers want to be aligned in memory, so if changing the format of this struct
-// keep the 1-byte 'Argnum' as the final parameter before the padding (if padding is needed).
-type Context struct {
-	Ts              uint64
+// EventContext contains common metadata that is collected for all types of events.
+//
+// NOTE:
+// - Use pahole to ensure this struct reflects the `event_context“ struct in the eBPF code.
+// - Update the `EbpfDecoder` if any change to this struct is done.
```

Comment:
The all CAPITAL WORDS for the "PLEASE NOTE" should remain. Its a very important statement made in other places as well (I remember reviewing that PR and it was a point of discussion, so lets keep maximuma attention by keepig A HUGE COMMENT SAYING THAT, please.

Reply:
OK, I will convert them back to CAPITAL.